pkg/controllers/jobset_controller.go

```diff
@@ -45,8 +45,9 @@ const (
 )
 
 var (
-	jobOwnerKey = ".metadata.controller"
-	apiGVStr    = jobset.GroupVersion.String()
+	jobOwnerKey                        = ".metadata.controller"
+	apiGVStr                           = jobset.GroupVersion.String()
+	JobConditionReasonPodFailurePolicy = "PodFailurePolicy"
```

Comment:
Looking at this again I think you want this to be a constant and not a var.  

I would also like to see a comment about why this is defined as is.  If someone changes the job api this would break and it would be confusing to find why this broke.  So I say comment to say a headache in the future.

Reply:
Makes sense, updated.